cscs-checks/microbenchmarks/cpu/likwid/likwid.py

```diff
@@ -8,85 +8,81 @@
 
 
 class MemBandwidthTest(rfm.RunOnlyRegressionTest):
-    def __init__(self):
-        self.modules = ['likwid']
-        self.valid_prog_environs = ['PrgEnv-gnu']
-        self.sourcesdir = None
+    modules = ['likwid']
+    valid_prog_environs = ['PrgEnv-gnu']
+    sourcesdir = None
+    executable = 'likwid-bench'
+    num_tasks = 1
+    num_tasks_per_node = 1
+    num_tasks_per_core = 2
+    system_num_cpus = {
+        'daint:mc':  72,
+        'daint:gpu': 24,
+        'dom:mc':  72,
+        'dom:gpu': 24,
+    }
```

Comment:
There is no need to set these statically anymore; they can be retrieved from `self.current_partition.processor.num_cpus`.